src/data/objects/signpost_dialogs.asm

```diff
@@ -0,0 +1,261 @@
+; Dialog ids for overworld signposts, indexed by hMapRoom.
+;
+; Only the lowest byte is stored. The dialog table is determined by
+; the calling code.
+SignpostDialogTable::
+._00 db_dialog_low Dialog10C ; unused
+._01 db_dialog_low Dialog10C ; unused
+._02 db_dialog_low Dialog10C ; unused
+._03 db_dialog_low Dialog10C ; unused
+._04 db_dialog_low Dialog10C ; unused
+._05 db_dialog_low Dialog10C ; unused
+._06 db_dialog_low Dialog10C ; unused
+._07 db_dialog_low Dialog10C ; unused
+._08 db_dialog_low Dialog10C ; unused
+._09 db_dialog_low Dialog10C ; unused
+._0A db_dialog_low Dialog10C ; unused
+._0B db_dialog_low Dialog10C ; unused
+._0C db_dialog_low Dialog10C ; unused
+._0D db_dialog_low Dialog10C ; unused
+._0E db_dialog_low Dialog10C ; unused
+._0F db_dialog_low Dialog10C ; unused
+._10 db_dialog_low Dialog10C ; unused
+._11 db_dialog_low Dialog1B6 ;
```

Comment:
What are the blanks? Unknown?

Reply:
The blanks are the values actually used - but I didn't have the courage to add a summary of each signpost in the game 😅